source/includes/_tipser-react-elements.md

```diff
@@ -0,0 +1,263 @@
+import { TipserElement, TipserProduct, CartIcon, TipserElementsProvider, ShopComponent } from '@tipser/tipser-elements';
+
+const tipserElementsConfig = {
+    lang: 'en',
+    primaryColor: 'blue',    
+};
+
+ReactDOM.render(
+    <TipserElementsProvider posId="59e86b79b8f3f60a94ecd26a" config={tipserElementsConfig}>   
+        <header>
+            <nav>
+                <span>Nav Element</span>
+                <CartIcon/>
+            </nav>
+        </header>
+        <main>
+            <TipserElement  id="1larHZb8TeMQiqmi4W8CIS" />
+            <TipserProduct productId="5ba2334a781baa0001ccdffc"/>
+            <ShopComponent />
+        </main>
+        <footer>
+            <span>This is the footer</span>
+        </footer>
+    </TipserElementsProvider>, 
+    document.getElementById('root'));
+```
+> "root" is the id of the HTML element where the Tipser element goes
+
+#### `TipserElementsProvider` 
+Entry point to Tipser Elements (creating a context for other Elements);
+
+prop name  | description | type  | required | default value 
+-----------|-------------|-------|----------|--------------
+posId | id of Point of sale | string | true | 
+config | configuration object (see [definition here](#all-configuration-options-of-tipser-elements)) | {} | false   | {}
+ 
+#### `TipserElement` 
+Generic Element that can render any Contentful content that's fed as a prop to the element.
+
+#### `TipserProduct` 
+Element that renders the product given the Tipser product ID as the prop.
+
+#### `CartIcon` 
+Element that displays the number of items in your cart and gives the user a way to open the checkout dialog.
+
+
+
+### All configuration options of Tipser Elements
+
+Configuration is an object, which should be inserted into `TipserElementsProvider` as `config` prop.
+
+All properties are optional:
+
+property name | description |type | default value
+--------------|-------------|-----|---------------
+`defaultAddedToCartPopup` | controls whether default Added To Cart Popup is displayed | boolean | true 
+`eventsHandlers` | gives possibility to add handlers to Tipser Elements events | object | {}
+`useDefaultErrorHandler` | when set to false and error happens, default message won't be displayed | boolean | true
+
+### Event Handlers
+```js
+
+let tipserConfig = {
+    lang: 'en',
+    primaryColor: 'red',
+    // ----- EVENT HANDLING START
+    useDefaultErrorHandler: true,
+    eventsHandlers: {
+        onError: (error) => {
+            console.log(error)
+        },
+        onAddToCart: ({cartSize, product}) => {
+            console.log('Hurray, you have added item to cart. ', product);
+            console.log('Your cart size is now. ', cartSize);
+        }
+    }
+    // -- EVENT HANDLING END
+};
+
+// ... as in example above class App extends Component { ... }
+
+```
+Event handlers may be passed as a part of `config` option of `TipserElementsProvider`. There is a number of events exposed for developer. 
```

Comment:
I know you just copied this section but I think it contains a lot of unnecessary verbose sentences, like " there is a number of events exposed for developer" or "Main goal of this handler is to add additional behavior when the error appears." (what is the secondary goal then? :) ), " when product is being added to cart event is triggered" (is this sentence even gramatically correct).

I can rewrite this section when you merge those changes.

Reply:
Yes, would be good to review this, I also was a bit lost in how is that supposed to work...